2Homework25.02.22/Trie/Trie.Tests/TestTrie.cs

```diff
@@ -0,0 +1,75 @@
+using NUnit.Framework;
+
+namespace Trie.Tests;
+
+public class Tests
+{
+
+    [Test]
+    public void TestTrieAddAndSize()
+    {
+        Trie trie = new Trie();
+        bool result = trie.Size == 0;
+        trie.Add(new string("cat"));
```

Comment:
```suggestion
        trie.Add("cat");
```
"cat" и так string, тут Вы фактически копию строки создаёте.